src/services/triggers/hooks/providers.hook.ts

```diff
@@ -26,8 +63,8 @@ export default {
     all: [],
     find: [],
     get: [],
-    create: [],
-    update: [],
+    create: [updatePlans],
```

Comment:
We should move this update from hook directly to the event handler because when we do precache we use service directly without hooks, in this case, we do not retrieve plans

Reply:
Alternatively, we could add a hook to precache?

Reply:
The model instance contains a reference to the sequelize instance.